src/test/java/com/redhat/cloud/policies/app/UserConfigServiceTest.java

```diff
@@ -75,6 +81,18 @@ void passNotificationResponse() {
         }
     }
 
+    @Test
+    void testFedramp() {
+        when(environmentFlags.isFedramp()).thenReturn(Boolean.FALSE);
```

Comment:
Ah.. this test is actually wrong... `mvnw clean test` don't execute it but `mvnw clean verify` does :facepalm:

Reply:
shouldn't test execute it as well?

Reply:
`mvn test` does not run integration tests. I had to run `mvn integration-test` or `mvn install|verify`

Reply:
ah. That's differnt to Gradle. Thanks for the hint :)